src/UI/Canvas.cs

```diff
@@ -0,0 +1,162 @@
+using TileMapper.Windowing;
+using Raylib_cs;
+using ImGuiNET;
+using System.Numerics;
+
+namespace TileMapper {
+    
+    public class Canvas : GraphicsWindow {
```

Comment:
Brackets should be like:
if (x)
{
// Code here.
}
Make sure you are following the C# guidelines.